app/src/component/AppNavigator.js

```diff
@@ -66,6 +66,7 @@ const discoverStack = createStackNavigator({
     navigationOptions: ({ navigation }) => ({
       title: 'Discover',
       headerLeft: menuNavigationButton(navigation),
+      headerTitleStyle: discoverStyle.titleText
```

Comment:
Is there a way to default to this? Instead of passing it in every time.

Reply:
Unfortunately not. Have to define custom navigation options for every navigator object created.

Reply:
You could have a default nav options object,

``
navigationOptions: ({ navigation }) => ({ ...defaultOptions, title: ... })
```

Reply:
I suppose this could work.